Formula/luyten.rb

```diff
@@ -28,10 +28,14 @@ def install
   end
 
   test do
-    io = IO.popen("luyten")
-    sleep 5
-    Process.kill("SIGINT", io.pid)
-    Process.wait(io.pid)
-    io.read !~ /Exception/
+    begin
+      io = IO.popen("#{bin}/luyten 2>&1")
+      sleep 5
+    ensure
+      Process.kill("SIGINT", io.pid)
+      Process.wait(io.pid)
+    end
+    output = io.read
+    assert_no_match /Exception/, output if output !~ /HeadlessException/
```

Comment:
This doesn't look particularly elegant or easy to understand. Is the idea that if a `HeadlessException` occurs, it should be silently swallowed, but if it doesn't there should also be no other exceptions?